pdal/SpatialReference.cpp

```diff
@@ -490,25 +505,18 @@ std::string SpatialReference::getWKT1() const
         return wkt;
 
     OGRScopedSpatialReference srs = ogrCreateSrs(wkt);
-    std::string wkt1;
-    if (srs)
-    {
-        char *buf = nullptr;
-        const char* apszOptions[] =
-            { "FORMAT=WKT1_GDAL", "ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES", nullptr };
-
-        srs->exportToWkt(&buf, apszOptions);
-        if (buf)
-        {
-            wkt1 = buf;
-            CPLFree(buf);
-        }
-    }
+    std::string wkt1 = exportToWkt(srs.get(), {"FORMAT=WKT1_GDAL", "ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES"});
```

Comment:
`"ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES"` should probably be controllable now that I look at it.

Reply:
This was mostly a note to future-hobu to add support somehow for user-controllability of overrides like this. No need to address it just now unless it matters to you.